account_move_name_sequence/views/account_journal_views.xml

```diff
@@ -16,15 +16,19 @@
                 <field
                     name="sequence_id"
                     required="1"
-                    context="{'default_name': name, 'default_company_id': company_id, 'default_implementation': 'no_gap', 'default_padding': 4, 'default_use_date_range': True, 'default_prefix': (code or 'UNKNOWN') + '/%%(range_year)s/'}"
+                    context="{'default_name': name, 'default_company_id': company_id, 'default_implementation': 'no_gap',
```

Comment:
One per line and breaking after braces, as black would format it.:

```suggestion
                    context="{
                        'default_name': name,
                        'default_company_id': company_id,
                        'default_implementation': 'no_gap',
```

Same below.